rest_framework/fields.py

```diff
@@ -1606,8 +1606,8 @@ def __init__(self, model_field, **kwargs):
         max_length = kwargs.pop('max_length', None)
         super(ModelField, self).__init__(**kwargs)
         if max_length is not None:
-            message = self.error_messages['max_length'].format(max_length=max_length)
-            self.validators.append(MaxLengthValidator(max_length, message=message))
+            message = self.error_messages['max_length']
+            self.validators.append(MaxLengthValidator(max_length, message=message, string_format='{'))
```

Comment:
@tomchristie move all the messages not to use the new format style *but* it will impact users that did customize it.